app/controllers/pets_controller.rb

```diff
@@ -16,7 +16,7 @@ def new
   end
 
   def create
-    pet = Pet.new(pet_params)
+    pet = Pet.create(pet_params)
```

Comment:
This should be kept as .new + .save! .create does both of the but when we want to capture any errors that can happen when saving, we break them up into `.new` and `.save` in order to capture the error case where a record isn't created properly and therefore can't be saved